sdk/maps/azure-maps-render/src/test/java/com/azure/maps/render/TestUtils.java

```diff
@@ -70,8 +82,10 @@ static Copyright getExpectedCopyrightFromBoundingBox() throws IOException {
 
     static Copyright getExpectedCopyrightForTile() throws IOException {
         InputStream is = ClassLoader.getSystemResourceAsStream("getcopyrightfortile.json");
+        File file = new File("getcopyrightfortileoutput.txt");
+        Files.copy(is, file.toPath(), StandardCopyOption.REPLACE_EXISTING);
         byte[] data = null;
-        data = is.readAllBytes();
+        data = Files.readAllBytes(file.toPath());
```

Comment:
where is data declared, by the way? It seems like it's a class variable for some reason, and that is not correct.